tests/kani/Intrinsics/bitreverse.rs

```diff
@@ -4,10 +4,6 @@
 // Check that we get the expected results for the `bitreverse` intrinsic
 // https://doc.rust-lang.org/std/intrinsics/fn.bitreverse.html
 
-// Note: Support for `__builtin_bitreverse` in CBMC is being added in
-// https://github.com/diffblue/cbmc/pull/6581
-// Tracking issue: https://github.com/model-checking/kani/issues/778
```

Comment:
I guess this might not belong in this test case if there's nothing to change here, but this work remains to be done, right?

Reply:
Not sure what you mean, we are enabling this test because support for `__builtin_bitreverse` is now available in CBMC. The Kani issue will be resolved by this PR, note that most of the work on Kani's end was added in #779

Reply:
> we are enabling this test because support for __builtin_bitreverse is now available in CBMC. 

Ah, that's what I was missing. ok.